Tools/scripts/mttr-build-ci.sh

```diff
@@ -26,8 +26,21 @@ echo $PATH
 ./waf configure --board=sitl --debug --enable-opendroneid
 ./waf copter
 
-mv build/MttrCubeBlack/bin/arducopter.apj "/tmp/deploy_files/copter-MttrCubeBlack.apj"
-mv build/MttrCubeOrange/bin/arducopter.apj "/tmp/deploy_files/copter-MttrCubeOrange.apj"
-mv build/MttrCubeBlack/bin/arducopter "/tmp/deploy_files/copter-MttrCubeBlack.elf"
-mv build/MttrCubeOrange/bin/arducopter "/tmp/deploy_files/copter-MttrCubeOrange.elf"
+# Get Ardupilot version
+eval $(sed -n 's/^#define  *\([^ ]*\)  *\(.*\) *$/export \1=\2/p' ArduCopter/version.h)
+
+# Get mttr git tag (replace all dashes with underscores)
+git fetch --tags --no-recurse-submodules
+MTTR_GIT_TAG=$(echo $(git tag -l "mttr-*" --sort=-committerdate | head -1) | tr - _)
+
+# Get mttr git hash
+MTTR_GIT_HASH=$(git rev-parse --short=7 HEAD)
```

Comment:
When this gets run is very important. It must be run after the commit gets merged to the main for this to work. If you can run a test that shows the hash is correct we can leave this in. Otherwise it will cause way more confusion than its worth.

Reply:
You are right. I ensured that it worked well on my pc, but I'm not sure it was right from circleci. I'm looking into this.

Reply:
@robkarol Here's the latest push with correct hash in the files: https://github.com/matternet/ardupilot/actions/runs/4008770308
<img width="538" alt="image" src="https://user-images.githubusercontent.com/2285455/214675649-d7765b48-9d21-4dd4-9701-fe02005178a7.png">